lib/cogctl/bundle_registry.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Cogctl.BundleRegistry do
```

Comment:
Maybe we should have an API in Cog that we hit to install the bundle instead, since this is duplicated from Cog.